docs/src/main/paradox/additional/operations.md

```diff
@@ -55,11 +55,4 @@ From JMX you can:
  * mark any node in the cluster as down
  * tell any node in the cluster to leave
 
-Member nodes are identified by their address, in format *`akka://actor-system-name@hostname:port`*.
-
-## Monitoring and Observability
```

Comment:
same here, Lightbend offering. Have we reached out to Kamon, or have they approached us? we could mention them there maybe 🤔